lib/libsys/open.2

```diff
@@ -195,6 +195,23 @@ error if file is not a directory
 .It Dv O_CLOEXEC
 automatically close file on
 .Xr execve 2
+.It Dv O_CLOFORK
```

Comment:
I suggest to move this long description into a dedicated paragraph in the main body of the man page.  There, you would leave a short explanation like
.It Dv O_CLOFORK
automatically close file on fork in the child process

Then the addition would follow the existing structure of the open man page: the list of flags gives the enumeration and overview, and detailed description is provided in the body.

Reply:
Hopefully fixed.

I must say you're a tough, thorough & patient reviewer. Thanks!